src/components/Carousel.js

```diff
@@ -636,6 +638,11 @@ export default class Carousel extends Component {
    * @return {ReactElement} element
    */
   renderArrowRight = () => {
+    const { children } = this.props;
```

Comment:
Slides are retrieved correctly by this.getChildren method so there no need to for this line. Don't access `children` directly this way.